client/my-sites/sidebar/sidebar.jsx

```diff
@@ -584,14 +575,12 @@ export class MySitesSidebar extends Component {
 
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
-			<Button
-				borderless
-				className="my-sites-sidebar__add-new-site"
-				href={ this.getAddNewSiteUrl() }
-				onClick={ this.focusContent }
-			>
-				<Gridicon icon="add-outline" /> { this.props.translate( 'Add New Site' ) }
-			</Button>
+			<SidebarItem
+				label={ this.props.translate( 'Add new site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '+ Add New Site' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/sidebar-CSS-refactor)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).